mxnet/inference/docker/1.7.0/py3/Dockerfile.cpu

```diff
@@ -0,0 +1,129 @@
+    LC_ALL=C.UTF-8 \
+    TEMP=/home/model-server/tmp
+
+RUN apt-get update \
+ && apt-get -y install --no-install-recommends \
+    build-essential \
+    ca-certificates \
+    curl \
+    emacs \
+    git \
+    libopencv-dev \
+    openjdk-8-jdk-headless \
+    vim \
+    wget \
+    zlib1g-dev \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+RUN wget -c https://www.openssl.org/source/openssl-${OPENSSL_VERSION}.tar.gz \
+ && apt-get update \
+ && apt remove -y --purge openssl \
+ && rm -rf /usr/include/openssl \
+ && apt-get install -y \
+    ca-certificates \
+ && tar -xzvf openssl-${OPENSSL_VERSION}.tar.gz \
+ && cd openssl-${OPENSSL_VERSION} \
+ && ./config && make && make test \
+ && make install \
+ && ldconfig \
+ && cd .. && rm -rf openssl-*
+
+# when we remove previous openssl, the ca-certificates pkgs and its symlinks gets deleted
+# causing sslcertverificationerror the below steps are to fix that
+RUN ln -s /etc/ssl/certs/*.* /usr/local/ssl/certs/
+
+RUN apt-get update \
+ && apt-get -y install --no-install-recommends \
+    openjdk-8-jdk \
+    openjdk-8-jre \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+RUN wget https://www.python.org/ftp/python/$PYTHON_VERSION/Python-$PYTHON_VERSION.tgz \
+ && tar -xvf Python-$PYTHON_VERSION.tgz \
+ && cd Python-$PYTHON_VERSION \
+ && ./configure \
+ && make \
+ && make install \
+ && apt-get update \
+ && apt-get install -y --no-install-recommends \
+    libreadline-gplv2-dev \
+    libncursesw5-dev \
+    libssl-dev \
+    libsqlite3-dev \
+    tk-dev \
+    libgdbm-dev \
+    libc6-dev \
+    libbz2-dev \
+ && make \
+ && make install \
+ && rm -rf ../Python-$PYTHON_VERSION* \
+ && ln -s /usr/local/bin/pip3 /usr/bin/pip
+
+RUN ln -s $(which ${PYTHON}) /usr/local/bin/python
+
+RUN ${PIP} --no-cache-dir install --upgrade \
+    pip \
+    setuptools
+
+WORKDIR /
+
+RUN ${PIP} install --no-cache-dir \
+    ${MX_URL} \
+    gluonnlp==0.10.0 \
+    gluoncv==0.6.0 \
+    multi-model-server==$MMS_VERSION \
+    keras-mxnet==2.2.4.1 \
+    numpy==1.17.4 \
+    onnx==1.4.1 \
+    "sagemaker-mxnet-inference<2"
+
+# This is here to make our installed version of OpenCV work.
+# https://stackoverflow.com/questions/29274638/opencv-libdc1394-error-failed-to-initialize-libdc1394
+# TODO: Should we be installing OpenCV in our image like this? Is there another way we can fix this?
+RUN ln -s /dev/null /dev/raw1394
+
+RUN useradd -m model-server \
+ && mkdir -p /home/model-server/tmp \
+ && chown -R model-server /home/model-server
+
+COPY mms-entrypoint.py /usr/local/bin/dockerd-entrypoint.py
+COPY config.properties /home/model-server
+
+RUN chmod +x /usr/local/bin/dockerd-entrypoint.py
+
+ADD https://raw.githubusercontent.com/aws/deep-learning-containers/master/src/deep_learning_container.py /usr/local/bin/deep_learning_container.py
+
+RUN chmod +x /usr/local/bin/deep_learning_container.py
+
+RUN curl https://aws-dlc-licenses.s3.amazonaws.com/aws-mxnet-1.6.0/license.txt -o /license.txt
```

Comment:
does this need to be updated also?

Reply:
@saimidu @arjkesh any idea?

Reply:
Yes @tejaschumbalkar is updating the license file I provided for 1.7.0.

Reply:
New license is available here: https://aws-dlc-licenses.s3.amazonaws.com/aws-mxnet-1.7.0/license.txt

Reply:
To make sure we comply with the new package naming, is it fine if you move the license to folder name `aws-mx-1.7.0`
We want to make sure we don't use "mxnet" for the package name or license.

Reply:
License moved to https://aws-dlc-licenses.s3.amazonaws.com/aws-mx-1.7.0/license.txt